buyer_data/models.py

```
@@ -1,7 +1,8 @@
-from django.db import models
+from django.db import models, transaction
```

Ruff (0.11.9)

`django.db.transaction` imported but unused

Remove unused import: `django.db.transaction`

(F401)